### PR TITLE
Add a script that can create a new user, upload files, and check for f…

### PR DESCRIPTION
**Pull request by mvdbeek**
…ile presence in galaxy's FTP upload form.

If we put this script in a cronjob we will be sure that ftp error reports are not due to password, commandline or other problems.

### CODE REVIEW

File: helper_scripts/test_upload.py [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+    args = parse_args()
+    web_protocol, fqdn, ftp_protocol, port, api_key = \
+    args.protocol, args.fqdn, args.ftp_protocol, args.port, args.api_key
+    if args.http_auth_username:
+        GalaxyInstance.make_delete_request = inject_auth(make_delete_request, args.http_auth_username, args.http_auth_password)
+        GalaxyInstance.make_post_request = inject_auth(make_post_request, args.http_auth_username, args.http_auth_password)
+        GalaxyInstance.make_put_request = inject_auth(make_put_request, args.http_auth_username, args.http_auth_password)
+        GalaxyInstance.make_get_request = inject_auth(GalaxyInstance.make_get_request, args.http_auth_username, args.http_auth_password)
+    url = web_protocol + "://" + fqdn
+    new_api_key, user_email, password = create_user(url, api_key)
```
Comment (drosofff) [inline helper_scripts/test_upload.py]: Please, can you avoid this step being systematic ?

Comment (mvdbeek) [inline helper_scripts/test_upload.py]: which step?

Comment (drosofff) [inline helper_scripts/test_upload.py]: new_api_key, user_email, password = create_user(url, api_key) :smile:

Comment (mvdbeek) [inline helper_scripts/test_upload.py]: why? I need it later?!

Comment (drosofff) [inline helper_scripts/test_upload.py]: yes... well you got the idea....
The bot is the bot. If we need to change its identity, we'll do it. But in standard days, I propose that the bot is a stable "person".

**Comment by drosofff** [comment]
@mvdbeek can you add a short description in the script header ?
Ready to merge but not favorable to have the script as a cron job

**Comment by drosofff** [comment]
@mvdbeek can you add a short description in the script header ?
I have merged. But I am not favorable to have the script as a daily cron job and to systematically create a new user, because it is complexifying the server stats

**Comment by mvdbeek** [comment]
@drosofff thanks. yes, monthly is sufficient.

**Pull request merged**